clients/admin-ui/src/features/system/SystemRegisterSuccess.tsx

```diff
@@ -44,6 +46,12 @@ const SystemRegisterSuccess = ({ system, onAddNextSystem }: Props) => {
     dispatch(setActiveSystem(undefined));
 
     router.push("/system");
+
+    successAlert(
+      `You have successfully added ${allRegisteredSystems?.length} systems to your Data Map`,
+      `${allRegisteredSystems?.length} Systems successfully added to your Data Map`,
+      { isClosable: true }
+    );
```

Comment:
Are you sure this is the file you meant to change? As we've talked about before, somehow `SuccessPage` has come back, and that's the one that shows up at the end of the config wizard. I think we'd have to put this in both, at least until we figure out how to merge those two files back to one again (can you make a ticket for that?)

Reply:
Both of these files are the correct file - I'll add a screenshot to the PR comment, but this alert is meant to show on the `/system` or `/datamap` route  **after** successfully registering (a) system(s). I added the alert here after the re-route to `/system` as well as the `ScanResultsForm.tsx` after successfully registering systems and redirecting to either `/system` or `/datamap`.